tests/nvidiagpu/deploy-gpu-test.go

```diff
@@ -540,51 +546,50 @@ var _ = Describe("GPU", Ordered, Label(tsparams.LabelSuite), func() {
 					gpuOperatorDeployment.Definition.Name)
 			}
 
-			// Need to handle this differently if deployed form bundle
+			By("Get the CSV deployed in NVIDIA GPU Operator namespace")
+			csvBuilderList, err := olm.ListClusterServiceVersion(inittools.APIClient, nvidiaGPUNamespace)
+
+			if err != nil {
+				glog.V(gpuparams.GpuLogLevel).Infof("Error getting list of CSVs in namespace '%s': %v",
+					nvidiaGPUNamespace, err)
+			}
+
+			Expect(err).ToNot(HaveOccurred(), "Error getting list of CSVs in GPU operator "+
+				"namespace: '%v'", err)
+
+			// Get the first element in the list - should normally be just one
+			csvBuilder := csvBuilderList[0]
```

Comment:
Maybe we should protect ourselves from unexpected cases like
```
Expect(csvBuilderList).To(HaveLen(1))
```
or at least
```
Expect(csvBuilderList).NotTo(BeEmpty())
```

Reply:
Good suggestion,  Done.